apps/helpers.py

```diff
@@ -9,6 +9,7 @@
 from .serialize import serialize_app
 from .tasks import deploy_resource
 
+from projects.models import Flavor
 ReleaseName = apps.get_model(app_label=settings.RELEASENAME_MODEL)
```

Comment:
Seems like you didn't apply pre-commit hooks, it would have formatted this

Reply:
Done!